src/modules/scraper/adapter/messaging/MerkleDistributorBlocksEventsConsumerV2.ts

```diff
@@ -25,38 +25,82 @@ export class MerkleDistributorBlocksEventsConsumerV2 {
     @InjectRepository(MerkleDistributorWindow)
     private merkleDistributorWindowRepository: Repository<MerkleDistributorWindow>,
     private appConfig: AppConfig,
+    private dataSource: DataSource,
   ) {}
 
   @Process({ concurrency: 1 })
   private async process(job: Job<MerkleDistributorBlocksEventsQueueMessage>) {
     const { chainId, from, to } = job.data;
-    const address = this.appConfig.values.web3.merkleDistributorContracts.opRewards.address;
-    const claimedEvents = (await this.providers
-      .getMerkleDistributorQuerier(chainId, address)
-      .getClaimedEvents(from, to)) as ClaimedEvent[];
-    this.logger.log(`(${from}, ${to}) - chainId ${chainId} - found ${claimedEvents.length} ClaimedEvent`);
-
-    for (const event of claimedEvents) {
-      try {
-        const claim = await this.fromClaimedEventToMerkleDistributorClaim(event, chainId, address);
-        await this.merkleDistributorClaimRepository.insert(claim);
-        this.scraperQueuesService.publishMessage<MerkleDistributorClaimQueueMessage>(
-          ScraperQueue.MerkleDistributorClaim,
-          {
-            claimId: claim.id,
-          },
-        );
-      } catch (error) {
-        if (error instanceof QueryFailedError && error.driverError?.code === "23505") {
-          // Ignore duplicate key value violates unique constraint error.
-          this.logger.warn(error);
-        } else {
```

Comment:
we don't need to handle this anymore?

Reply:
no, because this queryBuilder has this nice feature [orIgnore()](https://github.com/across-protocol/scraper-api/pull/458/files#diff-2dd0f7e424c3af2b19a2f54b3d0823d8027b4ec481fc0b060aa49cef4ce33af7R92). If the insert query breaks a table constraint, it won't do the insert